test/test_ops.py

```diff
@@ -1887,12 +1907,40 @@ def test_fake_autocast(self, device, dtype, op):
         context = torch.cuda.amp.autocast if device == "cuda" else torch.cpu.amp.autocast
         self._test_fake_helper(device, dtype, op, context)
 
+    @onlyCUDA
+    @ops([op for op in op_db if op.supports_autograd], allowed_dtypes=(torch.float,))
+    @skipOps('TestFakeTensor', 'test_fake_crossref_backward', fake_backward_xfails)
+    def test_fake_crossref_backward(self, device, dtype, op):
+        # tests fake tensor property propagation through a cross ref mode
+        # on ops which support backward
+        samples = op.sample_inputs(device, dtype, requires_grad=True)
+
+        for iter, sample in enumerate(samples):
+            args = [sample.input] + list(sample.args)
+            kwargs = sample.kwargs
+
+            # skip these to speed up tests
+            common_skip_ops = (
+                aten.detach.default,
+                aten.empty_strided.default,
+                aten.copy_.default,
+                aten.is_same_size.default,
+
+            )
+            # TODO: enable check_aliasing, too many failures :/
```

Comment:
ALL of the primtorch metas alias incorrectly, they all need to be fixed

Reply:
Well, they're not registered to fake tensor right now..

Reply:
Are you sure? I thought we hit them for dynamic shapes preferentially

Reply:
We do, you're right, but they're disabled for static fake. + 1 motivation for regularizing the path symbolic/fake (not to mention https://github.com/pytorch/pytorch/issues/84308). I have a pr that i had been working on to do this that i havent quite been able to push through finish line.

The aliasing tests are on for the non backward. I'll work on getting them on for backward as a follow up.